packages/pure/src/device/device.types.ts

```diff
@@ -71,6 +71,7 @@ export enum Endpoint {
   Messages = 8,
   CallLog = 9,
   Security = 13,
+  Outbox = 13,
```

Comment:
Is the outbox endpoint have the same name as security endpoint?

Reply:
[c4dd030](https://github.com/mudita/mudita-center/pull/670/commits/c4dd03028534a46de10837e02f5a176e65a5903d)